packages/sdk/src/auth/services/storage-service.ts

```
@@ -1,128 +1,74 @@
-  private storage: IStorage;
+  private isBrowser: boolean;
 
-  constructor(storageImplementation?: IStorage) {
-    // If no storage implementation is provided, try to detect the environment
-    if (!storageImplementation) {
-      if (typeof window !== 'undefined' && window.localStorage) {
-        this.storage = new BrowserStorage();
-      } else {
-        this.storage = new MemoryStorage();
-      }
-    } else {
-      this.storage = storageImplementation;
-    }
+  constructor() {
+    this.isBrowser = typeof window !== 'undefined' && window.localStorage !== undefined;
   }
 
   /**
    * Store a JWT token
    * @param jwt - The JWT string to store
    */
   async storeJWT(jwt: string): Promise<void> {
-    await this.storage.setItem(Storage.JWT_KEY, jwt);
+    if (!this.isBrowser) {
+      console.warn('Storage operations are not supported in Node environment');
+      return;
+    }
+    try {
+      localStorage.setItem(Storage.JWT_KEY, jwt);
+    } catch (error) {
+      console.error('Error storing JWT in localStorage:', error);
+      throw error;
+    }
   }
 
   /**
    * Retrieve the stored JWT token
    * @returns The stored JWT string or null if not found
    */
   async getJWT(): Promise<string | null> {
-    return await this.storage.getItem(Storage.JWT_KEY);
+    if (!this.isBrowser) {
+      console.warn('Storage operations are not supported in Node environment');
+      return null;
+    }
+    try {
+      return localStorage.getItem(Storage.JWT_KEY);
+    } catch (error) {
+      console.error('Error retrieving JWT from localStorage:', error);
+      throw error;
+    }
   }
 
   /**
    * Remove the stored JWT token (logout)
    */
   async clearJWT(): Promise<void> {
-    await this.storage.removeItem(Storage.JWT_KEY);
+    if (!this.isBrowser) {
+      console.warn('Storage operations are not supported in Node environment');
+      return;
+    }
+    try {
+      localStorage.removeItem(Storage.JWT_KEY);
+    } catch (error) {
```

:hocho: Just let the error bubble up; logging in catch statements, then re-throwing the error is an anti-pattern that almost always leads to excessive log output.  A good general rule-of-thumb is to only log errors when they are being handled, otherwise every level of your stack will end up logging and re-throwing the _same error_. 

This applies for all of the methods in this class that follow this pattern